packages/ui/pages/_lang/index.vue

```diff
@@ -1,31 +1,22 @@
 <template>
   <div>
-    <h2>Feeds</h2>
-    <div v-for="feed in feeds" :key="feed.id">
-      <p>{{ feed }}</p>
-    </div>
-    <h2>Feed</h2>
-    <p>{{ feed }}</p>
+    <Main />
   </div>
 </template>
 
 <script>
+import Main from '@/components/Main.vue'
```

Comment:
Remember we don't need to import components because we are importing them in `nuxt.config.js`
```suggestion
```